apps/backoffice-v2/src/common/hooks/useZodSearchParams/useZodSearchParams.tsx

```diff
@@ -11,8 +12,14 @@ export const useZodSearchParams = <TSchema extends AnyZodObject>(
   const parsedSearchParams = useMemo(() => schema.parse(searchParams), [schema, searchParams]);
 
   useEffect(() => {
-    setSearchParams(parsedSearchParams);
-  }, []);
+    const isSearchParamsEqual = checkIsLooselyEqualDeep(searchParams, parsedSearchParams);
+    console.log('test');
+    if (isSearchParamsEqual) {
+      return;
+    }
+
+    return setSearchParams(parsedSearchParams);
+  }, [parsedSearchParams, searchParams, setSearchParams]);
```

Comment:
Also fixes default search query params outside of the transaction monitoring scope.